service/test/NVMLIOGroupTest.cpp

```diff
@@ -241,6 +241,7 @@ TEST_F(NVMLIOGroupTest, read_signal)
 
     std::vector<double> mock_freq = {1530, 1320, 420, 135};
     std::vector<unsigned int> mock_supported_freq = {135, 142, 407, 414, 760, 882, 1170, 1530};
+    double step_size = 199.285714;
```

Comment:
Should the step size resolve to an integer value?

Reply:
This is meant to simulate setups where the step size might be variable (ex: V100s have an alternating 7MHz, 8MHz step size so 7.5Mhz is returned).  
We could change step size to an integer if that's preferable.